subsys/dfu/dfu_target/src/dfu_target_uart_cddl/dfu_target_uart.cddl

```diff
@@ -0,0 +1,32 @@
+;
+; Copyright (c) 2022 INTELLINIUM <giuliano.franchetto@intellinium.com>
+;
+
+; The envelop that will be prepended to the update binary.
+
+DFU_UART_target_Envelop = [
+	; The magic number needs to be set to the correct value for the
+	; correct target type to be chosen.
+	magic: uint,
+
+	; The remote target identifier.
+	; Used if multiple remote targets are
+	; available.
+	; This field is optional. If unset, it is assumed that the receiver
+	; of this envelop only has 1 remote target.
+	rem_target: uint .size 1,
```

Comment:
Doesn't look optional, `?` specifies if an element is optional

Reply:
comment fixed.